packages/components/src/components/domainproperties/DomainFormDisplayOptions.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2020 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+export interface IDomainFormDisplayOptions {
+    showRequired?: boolean
+    showValidators?: boolean
+    isDragDisabled?: boolean
+    showTextOptions?: boolean
+    phiLevelDisabled?: boolean
+}
+
+export class DomainFormDisplayOptions extends React.Component<IDomainFormDisplayOptions, any> {
```

Comment:
This isn't actually a react component is it? You are just trying to define the typescript object/type/interface, right? I would have expected this to just be the interface and then have a defaultProps constant to use in various places.

Reply:
I would probably just move this interface to the domainproperties/models.ts file instead of having a new file for it.